Setup.sh

```diff
@@ -65,6 +66,8 @@ spatial package retrieve worker_sdk  c_headers                           "${PINN
 spatial package retrieve worker_sdk  c-dynamic-x86_64-clang-macos        "${PINNED_CORE_SDK_VERSION}"  "${CORE_SDK_DIR}"/worker_sdk/c-dynamic-x86_64-clang-macos.zip
 spatial package retrieve worker_sdk  c-static-fullylinked-arm-clang-ios  "${PINNED_CORE_SDK_VERSION}"  "${CORE_SDK_DIR}"/worker_sdk/c-static-fullylinked-arm-clang-ios.zip
 spatial package retrieve worker_sdk  csharp                              "${PINNED_CORE_SDK_VERSION}"  "${CORE_SDK_DIR}"/worker_sdk/csharp.zip
+spatial package retrieve spot        spot-macos                          "${PINNED_SPOT_VERSION}"      "${BINARIES_DIR}"/Programs/spot
```

Comment:
So, @improbable-danny looks like we will need to change this file too to support China :D

Reply:
Already done :D